virttest/migration.py

```diff
@@ -59,40 +59,47 @@ def post_migration_check(self, vms, params, uptime=None, uri=None):
 
         :param vms: VM objects of migrating vms
         :param uptime: uptime dict of vms before migration
-        :param uri: target virsh uri
+        :param dest_uri: target virsh uri
+        :param src_uri: source virsh uri
         :return: updated dict of uptime
         """
-        vm_state = params.get("virsh_migrated_state", "running")
+        vm_state_dest = params.get("virsh_migrate_dest_state", "running")
+        vm_state_src = params.get("virsh_migrate_src_state", "shut off")
```

Comment:
Just a reminder.
In existing tp-libvirt cases, some python files already use 'virsh_migrate_src_state' for another purpose like the prepared vm state before migration. So we need to be careful if those files use this function in future and we need rename the variable if any. But so far so good.